tools/app/cypress.config.ts

```
@@ -1,28 +1,52 @@
 import { defineConfig } from 'cypress';
 import { execSync } from 'node:child_process';
-import { existsSync, rmSync } from 'node:fs';
+import { rmSync } from 'node:fs';
 
 // Path for test project that is created and modified during tests
 const batPath = '../../.tmp/cyberismo-bat';
 
 export default defineConfig({
   e2e: {
     baseUrl: 'http://localhost:3000',
+    numTestsKeptInMemory: 0,
+    experimentalMemoryManagement: true,
     setupNodeEvents(on) {
       on('task', {
         deleteTestProject() {
-          rmSync(batPath, { recursive: true, force: true });
-          return true;
+          try {
+            rmSync(batPath, {
+              recursive: true,
+              force: true,
+              maxRetries: 3,
+              retryDelay: 100,
+            });
+          } catch (error) {
+            console.error(
+              'Warning: When deleting test project - Failed to delete test project:',
+              error,
+            );
+          }
+          return null; // Return null instead of true to prevent EPIPE errors
         },
         createTestProject() {
-          if (existsSync(batPath)) {
-            rmSync(batPath, { recursive: true, force: true });
+          try {
```

Do we need to try catch? With `force` this won't fail if folder does not exist. If this still happens some day, the error is easily hidden as it ignores the error and continues to other tests, which will likely fail. By throwing the error, cypress can point to the failed task

---

Good point; since I am using `force` it won't fail --> bye bye `throw-catch`